proto/cosmos/store/streaming/abci/grpc.proto

```diff
@@ -7,34 +7,14 @@ import "cosmos/store/v1beta1/listening.proto";
 
 option go_package = "cosmossdk.io/store/streaming/abci";
 
-// ListenBeginBlockRequest is the request type for the ListenBeginBlock RPC method
-message ListenBeginBlockRequest {
-  tendermint.abci.RequestBeginBlock  req = 1;
-  tendermint.abci.ResponseBeginBlock res = 2;
-}
-
-// ListenBeginBlockResponse is the response type for the ListenBeginBlock RPC method
-message ListenBeginBlockResponse {}
-
 // ListenEndBlockRequest is the request type for the ListenEndBlock RPC method
-message ListenEndBlockRequest {
-  tendermint.abci.RequestEndBlock  req = 1;
-  tendermint.abci.ResponseEndBlock res = 2;
+message ListenFinalizeBlockRequest {
+  tendermint.abci.RequestFinalizeBlock  req = 1;
+  tendermint.abci.ResponseFinalizeBlock res = 2;
```

Comment:
Should we make these field numbers 3,4 respectively and comment out the old fields?

Reply:
i.e reserve 1,2

Reply:
since this is a new message type i dont think its needed